### PR TITLE
[SRVCOM-752] Add admin monitoring docs

### PR DESCRIPTION
**Pull request by abrennan89**

### CODE REVIEW

File: serverless/monitoring/serverless-monitoring-serving.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+. In the *Administrator* perspective, navigate to *Monitoring* -> *Dashboards*.
+. Select the *Knative Health Status* dashboard to view the overall health status of Knative Serving and Eventing.
++
+image::knative-health-status-dash.png[Knative Health Status dashboard]
+. Select the *Knative Serving - Revision CPU and Memory Usage* dashboard to view metrics for Knative service revisions.
++
+You can filter this dashboard by *Namespace*, *Configuration*, or *Revision*, by using the drop-down selections.
++
+image::knative-serving-cpu-memory-dash.png[Knative Serving Revision CPU and Memory Usage dashboard]
```
Comment (abrennan89) [inline serverless/monitoring/serverless-monitoring-serving.adoc:30]: Split into 2 procedures. Eventing docs coming later for #2.

**Comment by skonto** [comment]
@abrennan89 if this targets master and not a release, then I have updated lately the dashboards on the serverless-operator master branch (This will be in 1.13 afaik):

image
There is kafka support and the status reports readiness since health is not well defined yet.

Also there are some panel titles updates in the serving resource dashboard:
image

**Comment by abrennan89** [comment]
@skonto 
1. Will I go ahead and get this PR reviewed and merged for 1.12.0, or do you want to hold off until 1.13.0 and make the changes before we publish the docs?
2. Which OCP version(s) does this apply for?
3. If someone doesn't have Kafka installed, does this status show up anyway, or only if an admin installs it?

**Review by mgencur — COMMENTED** [review]
Overall, it looks good. But I think we already have monitoring available for Eventing as well - CPU/mem. Can you confirm this @skonto ?
This PR only mentions Serving.

**Comment by skonto** [comment]
@mgencur oin the 1.12 branch we have the following [dashboards](https://github.com/openshift-knative/serverless-operator/tree/release-1.12/knative-operator/deploy/resources/dashboards):
-  grafana-dash-knative-eventing-broker.yaml (eventing broker/filter)
- grafana-dash-knative-eventing-source.yaml (eventing sources)
- grafana-dash-knative-health.yaml (health status)
- grafana-dash-knative.yaml (serving resources, needs renaming)

Eventing resources comes in 1.13: https://github.com/openshift-knative/serverless-operator/tree/master/knative-operator/deploy/resources/dashboards
-  grafana-dash-knative-eventing-broker.yaml (eventing broker/filter)
- grafana-dash-knative-eventing-source.yaml (eventing sources)
- grafana-dash-knative-health.yaml (health status)
- grafana-dash-knative-serving-resources.yaml (serving resources)
- grafana-dash-knative-eventing-resources.yaml (eventing resources)

**Review by sfortner-RH — APPROVED** [review]
lgtm!

**Pull request merged**